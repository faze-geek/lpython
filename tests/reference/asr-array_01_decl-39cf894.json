{
    "basename": "asr-array_01_decl-39cf894",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/array_01_decl.py",
    "infile_hash": "a0894dbdcbfd15ade316e11aba63284d5f1a2b5c011417db64fab06d",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-array_01_decl-39cf894.stderr",
    "stderr_hash": "1e98330a7af8f015329d3574c2fb3c03429f07c6ef99996c82f1876a",
    "returncode": 2
}