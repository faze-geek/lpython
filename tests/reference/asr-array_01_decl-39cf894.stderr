semantic error: 'i16[:]' object does not support item assignment
  --> tests/../integration_tests/array_01_decl.py:10:5
   |
10 |     xi16[2] = i16(32)
   |     ^^^^^^^^^^^^^^^^^ 
